shared_crates/ui/src/scroll_area.rs

```diff
@@ -23,6 +27,8 @@ pub enum ScrollAreaSizing {
     FitChildrenWidth,
     /// Assumes the width from the parent and propagates it to the children
     FitParentWidth,
+    /// Fit to children width, height is screen and assign a max scroll down
```

Comment:
Hm I don't think this makes sense as an option to this enum, it's got nothing to do with how the width is adjusted.